CIT365_W9_MegaDeskV2/Pages/Shared/_Layout.cshtml

```diff
@@ -2,8 +2,8 @@
 <html lang="en">
 <head>
     <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>@ViewData["Title"] - CIT365_W9_MegaDeskV2</title>
+    <meta name="viewport" content="wIdth=device-wIdth, initial-scale=1.0" />
```

Comment:
upper case "I" in width

Reply:
huh! Great catch! I'm not even sure how that got in there!! Must have been typo in a naming refactor and I missed that when reviewing changes. Will fix